hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/tables/insights/deletePendingKeysTable.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import React from 'react';
```

Comment:
While attempting to create keys pending deletion, I noticed data duplication. For example, in the old UI, when you click the plus icon to expand and view details about a deleted key, the record displays correctly.

<img width="1161" alt="image" src="https://github.com/user-attachments/assets/b867a8f1-4f85-4b38-8945-ee868e62cedf">

However, in the new UI, the same record appears duplicated multiple times. Interestingly, this duplication does not occur in Safari when accessing the same link. Should we address this issue, or could it simply be related to a browser cache problem on my end?

<img width="1160" alt="image" src="https://github.com/user-attachments/assets/cd16cd1e-560d-4a9e-a235-c658bb5a4be9">

Reply:
Addressed in [3158153](https://github.com/apache/ozone/pull/7327/commits/31581534586372326b8d2387e53c159256348854)